scripts/bump-ios-sdk-version.sh

```diff
@@ -34,13 +34,16 @@ fi
 echo "React-Native iOS version $oldiOSSdkVersion will be updated to $newiOSSdkVersion"
 
 # Update iOS sdk version in postlink.js for appcenter-crashes, appcenter-analytics,
-# appcenter-push projects
+# appcenter-auth appcenter-push projects
 fileContent="$(cat ./appcenter-crashes/scripts/postlink.js)"
 echo "${fileContent/\'AppCenter\/Crashes\', version\: \'$oldiOSSdkVersion\'/'AppCenter/Crashes', version: '$newiOSSdkVersion'}" > ./appcenter-crashes/scripts/postlink.js
 
 fileContent="$(cat ./appcenter-analytics/scripts/postlink.js)"
 echo "${fileContent/\'AppCenter\/Analytics\', version\: \'$oldiOSSdkVersion\'/'AppCenter/Analytics', version: '$newiOSSdkVersion'}" > ./appcenter-analytics/scripts/postlink.js
 
+fileContent="$(cat ./appcenter-auth/scripts/postlink.js)"
+echo "${fileContent/\'AppCenter\/Analytics\', version\: \'$oldiOSSdkVersion\'/'AppCenter/Analytics', version: '$newiOSSdkVersion'}" > ./appcenter-auth/scripts/postlink.js
```

Comment:
```suggestion
echo "${fileContent/\'AppCenter\/Auth\', version\: \'$oldiOSSdkVersion\'/'AppCenter/Auth', version: '$newiOSSdkVersion'}" > ./appcenter-auth/scripts/postlink.js
```

Reply:
Done

Reply:
Thanks for catching this

Reply:
You don't need to modify locally for github suggestions, they can be applied directly (I already clicked apply).